### PR TITLE
CRED-168: Redesign login

### PR DESCRIPTION
**Pull request by zignd**
## Contexto

Atualização de design a nível de /account, nessa pull request mais especificamente no LoginForm e no Presentation. Estruturas base da página de Account também foram alteradas e essa alteração será utilizada também nas próximas iterações que irão alterar as outras páginas nesse nível.

## Issues linkadas
https://mundipagg.atlassian.net/browse/CRED-168

## Como testar?

Acessar localmente a rota `/#/account/login` com o environment definido como `live`.

### CODE REVIEW

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: a63cf112683897a79c92bf030619151c6bc9d468
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: a63cf112683897a79c92bf030619151c6bc9d468
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: a63cf112683897a79c92bf030619151c6bc9d468
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: a63cf112683897a79c92bf030619151c6bc9d468
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: a63cf112683897a79c92bf030619151c6bc9d468
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

 * Commit: 73c32b927230437537027cdca183322f8b17e0ee
  * Always use the standard `scope: action description`, always in english  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: a63cf112683897a79c92bf030619151c6bc9d468
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

 * Commit: 73c32b927230437537027cdca183322f8b17e0ee
  * Always use the standard `scope: action description`, always in english  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: a63cf112683897a79c92bf030619151c6bc9d468
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

 * Commit: 73c32b927230437537027cdca183322f8b17e0ee
  * Always use the standard `scope: action description`, always in english  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: a63cf112683897a79c92bf030619151c6bc9d468
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

 * Commit: 73c32b927230437537027cdca183322f8b17e0ee
  * Always use the standard `scope: action description`, always in english  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: a63cf112683897a79c92bf030619151c6bc9d468
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

 * Commit: 73c32b927230437537027cdca183322f8b17e0ee
  * Always use the standard `scope: action description`, always in english  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

 * Commit: 6025c40efc692a03eca7f0bd9eb39901ff10b829
  * Always use the standard `scope: action description`, always in english  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

 * Commit: 7e34900d33e5f0ae95f88ed39e3a28b5eef66de2
  * Always use the standard `scope: action description`, always in english  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

 * Commit: 17b8a84855aacf81723c70b55b7c7fee371323ba
  * Always use the standard `scope: action description`, always in english  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: a63cf112683897a79c92bf030619151c6bc9d468
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

 * Commit: 73c32b927230437537027cdca183322f8b17e0ee
  * Always use the standard `scope: action description`, always in english  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

 * Commit: 6025c40efc692a03eca7f0bd9eb39901ff10b829
  * Always use the standard `scope: action description`, always in english  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

 * Commit: 7e34900d33e5f0ae95f88ed39e3a28b5eef66de2
  * Always use the standard `scope: action description`, always in english  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

 * Commit: 17b8a84855aacf81723c70b55b7c7fee371323ba
  * Always use the standard `scope: action description`, always in english  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: a63cf112683897a79c92bf030619151c6bc9d468
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

 * Commit: 73c32b927230437537027cdca183322f8b17e0ee
  * Always use the standard `scope: action description`, always in english  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

 * Commit: 6025c40efc692a03eca7f0bd9eb39901ff10b829
  * Always use the standard `scope: action description`, always in english  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

 * Commit: 7e34900d33e5f0ae95f88ed39e3a28b5eef66de2
  * Always use the standard `scope: action description`, always in english  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

 * Commit: 17b8a84855aacf81723c70b55b7c7fee371323ba
  * Always use the standard `scope: action description`, always in english  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: a63cf112683897a79c92bf030619151c6bc9d468
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

 * Commit: 73c32b927230437537027cdca183322f8b17e0ee
  * Always use the standard `scope: action description`, always in english  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

 * Commit: 6025c40efc692a03eca7f0bd9eb39901ff10b829
  * Always use the standard `scope: action description`, always in english  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

 * Commit: 7e34900d33e5f0ae95f88ed39e3a28b5eef66de2
  * Always use the standard `scope: action description`, always in english  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

 * Commit: 17b8a84855aacf81723c70b55b7c7fee371323ba
  * Always use the standard `scope: action description`, always in english  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: a63cf112683897a79c92bf030619151c6bc9d468
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

 * Commit: 73c32b927230437537027cdca183322f8b17e0ee
  * Always use the standard `scope: action description`, always in english  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

 * Commit: 6025c40efc692a03eca7f0bd9eb39901ff10b829
  * Always use the standard `scope: action description`, always in english  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

 * Commit: 7e34900d33e5f0ae95f88ed39e3a28b5eef66de2
  * Always use the standard `scope: action description`, always in english  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

 * Commit: 17b8a84855aacf81723c70b55b7c7fee371323ba
  * Always use the standard `scope: action description`, always in english  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

 * Commit: 65031b1f8bc78eca56fbd6c7d79f1021f444a5c8
  * Always use the standard `scope: action description`, always in english  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: a63cf112683897a79c92bf030619151c6bc9d468
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

 * Commit: 73c32b927230437537027cdca183322f8b17e0ee
  * Always use the standard `scope: action description`, always in english  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

 * Commit: 6025c40efc692a03eca7f0bd9eb39901ff10b829
  * Always use the standard `scope: action description`, always in english  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

 * Commit: 7e34900d33e5f0ae95f88ed39e3a28b5eef66de2
  * Always use the standard `scope: action description`, always in english  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

 * Commit: 17b8a84855aacf81723c70b55b7c7fee371323ba
  * Always use the standard `scope: action description`, always in english  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

 * Commit: 65031b1f8bc78eca56fbd6c7d79f1021f444a5c8
  * Always use the standard `scope: action description`, always in english  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: a63cf112683897a79c92bf030619151c6bc9d468
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

 * Commit: 73c32b927230437537027cdca183322f8b17e0ee
  * Always use the standard `scope: action description`, always in english  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

 * Commit: 6025c40efc692a03eca7f0bd9eb39901ff10b829
  * Always use the standard `scope: action description`, always in english  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

 * Commit: 7e34900d33e5f0ae95f88ed39e3a28b5eef66de2
  * Always use the standard `scope: action description`, always in english  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

 * Commit: 17b8a84855aacf81723c70b55b7c7fee371323ba
  * Always use the standard `scope: action description`, always in english  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

 * Commit: 65031b1f8bc78eca56fbd6c7d79f1021f444a5c8
  * Always use the standard `scope: action description`, always in english  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: a63cf112683897a79c92bf030619151c6bc9d468
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

 * Commit: 73c32b927230437537027cdca183322f8b17e0ee
  * Always use the standard `scope: action description`, always in english  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

 * Commit: 6025c40efc692a03eca7f0bd9eb39901ff10b829
  * Always use the standard `scope: action description`, always in english  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

 * Commit: 7e34900d33e5f0ae95f88ed39e3a28b5eef66de2
  * Always use the standard `scope: action description`, always in english  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

 * Commit: 17b8a84855aacf81723c70b55b7c7fee371323ba
  * Always use the standard `scope: action description`, always in english  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

 * Commit: 65031b1f8bc78eca56fbd6c7d79f1021f444a5c8
  * Always use the standard `scope: action description`, always in english  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

File: packages/pilot/package.json [hunk 1]
```diff
@@ -41,6 +44,7 @@
     "redux-observable": "1.0.0",
     "rxjs": "6.4.0",
     "shortid": "2.2.14",
+    "styled-components": "^5.1.1",
```
Comment (rsmelo) [inline packages/pilot/package.json:47]: pessoal sera que é a melhor opção ja estar usando material ui e styled component nesse projeto? será que não seria melhor ter esses componentes em outro local pra não fugir de controle isso e até mesmo entender a necessidade de adicionar essas novas dependecias ao projeto?

Comment (eduardogspereira) [reply packages/pilot/package.json:47]: Oi @rsmelo, eh tranquilo. A gente fez essa pagina usando essas libs mais como uma exploração mesmo, mas a gente tem um card pra remover esses componentes e usar os componentes que implementarmos na biblioteca de design :+1:

Comment (rsmelo) [reply packages/pilot/package.json:47]: beleza Edu, acho legal a gente bater um papo sobre isso depois com mais contexto. Outro ponto, tomar cuidado com a dependecia @material-ui/lab ela não é estavel e pode ter break changes sem aviso previo https://material-ui.com/components/about-the-lab/

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 8ead8cdd4720c42bedbd3929423f83190358c671
  * Always use the standard `scope: action description`, always in english  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 8ead8cdd4720c42bedbd3929423f83190358c671
  * Always use the standard `scope: action description`, always in english  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 8ead8cdd4720c42bedbd3929423f83190358c671
  * Always use the standard `scope: action description`, always in english  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 8ead8cdd4720c42bedbd3929423f83190358c671
  * Always use the standard `scope: action description`, always in english  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by miglsoares** [comment]
:eyes:

File: packages/pilot/src/containers/Account/style.css [hunk 2]
```diff
@@ -2,220 +2,105 @@
...
-  color: var(--color-picklerick-green-100);
-  font-weight: bold;
-  outline: none;
-  text-decoration: none;
-  text-transform: uppercase;
+.interactionSideContent {
+  display: flex;
+  flex-direction: column;
+  align-items: stretch;
+  width: 361px;
```
Comment (rsmelo) [inline packages/pilot/src/containers/Account/style.css:66]: esse numero quebrado pode ser arredondado?

File: packages/pilot/src/containers/Account/ui/Alert.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
+import React from 'react'
```
Comment (rsmelo) [inline packages/pilot/src/containers/Account/ui/Alert.js:1]: o diretorio /ui dentro do containar foge do padrão esperado no projeto. 
Componentes basico de ui não deveriam ser desenvolvidos na pilot mas serem implementados no foremer-kit e consumidos pela pilot

File: packages/pilot/src/containers/Account/Presentation/style.css [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+  @media (max-width: 1170px) {
+    display: none;
+  }
+}
+
+.presentationSideTest {
+  background-color: var(--color-barney-purple-100);
+}
+
+.presentationSideContent {
```
Comment (rsmelo) [inline packages/pilot/src/containers/Account/Presentation/style.css:23]: talvez uma alterativa pra esse content que usa apenas coisas de Flexbox seria usar o componente `Flexbox` do former-kit, é apenas uma sugestão =)

File: packages/pilot/src/containers/Account/Presentation/style.css [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+  }
+}
+
+.content > div {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 570px;
+  text-align: center;
+}
```
Comment (rsmelo) [inline packages/pilot/src/containers/Account/Presentation/style.css:55]: esse é ooutro que talvez desse de usar o componente Flexbox, apenas sugestão

File: packages/pilot/src/containers/Account/style.css [hunk 6]
```diff
@@ -2,220 +2,115 @@
...
-  max-width: 375px;
-  width: 100%;
-}
-
-.primaryContent form {
-  width: 100%;
+.forgotPassword {
+  align-self: flex-end;
+  font-size: 14px;
+  margin-top: 8px;
```
Comment (rsmelo) [inline packages/pilot/src/containers/Account/style.css:14]: poderia usar --spacing-tiny

**Pull request merged**